store/tikv/oracle/oracles/pd.go

```diff
@@ -231,3 +231,32 @@ func (o *pdOracle) GetLowResolutionTimestampAsync(ctx context.Context, opt *orac
 		err: nil,
 	}
 }
+
+func (o *pdOracle) getStaleTimestamp(ctx context.Context, prevSecond int64) (uint64, error) {
+	ts, ok := o.getLastTS("")
+	if !ok {
+		return 0, errors.Errorf("get stale timestamp, invalid txnScope = %s", oracle.GlobalTxnScope)
+	}
+
+	physicalTime := oracle.GetTimeFromTS(ts)
+	if physicalTime.Unix() <= prevSecond {
+		return 0, errors.Errorf("get invalid prevSecond, prevSecond must less than physicalTime, "+
+			"get prevSecond: %d, physicalTime: %d", prevSecond, physicalTime.Unix())
+	}
+
+	if prevSecond < 2 {
+		logutil.Logger(ctx).Warn("prevSecond may inaccurate when it less than 2",
+			zap.Int64("prev Second", prevSecond))
+	}
```

Comment:
This is unnecessary.

Reply:
Thanks for your review, I agree with you.

Reply:
Thanks for your review, I agree with you.